.install_parallel_netcdf.sh

```diff
@@ -4,19 +4,21 @@
 # broken on most distributions.
 #
 
-curl -k https://parallel-netcdf.github.io/Release/pnetcdf-${PNETCDF_VERSION}.tar.gz | tar -xzC ${BUILDDIR} &&
-  cd ${BUILDDIR}/pnetcdf-${PNETCDF_VERSION} &&
-  CC=mpicc CXX=mpicxx ./configure --disable-fortran --disable-cxx --enable-shared --prefix=${PREFIX} &&
-  make &&
-  make install &&
-  cd -
+# Installed via apt in .travis.yml
```

Comment:
so the  apt-packages libpnetcdf-dev libhdf5-openmpi-dev are working properly now, only NetCDF needs to be compiled ourselves ? Is that generally true ?

Reply:
No... it does not work, I reverted that change